src/constructs/autoscaling/asg.ts

```diff
@@ -67,6 +66,7 @@ export class GuAutoScalingGroup extends GuStatefulMigratableConstruct(GuAppAware
       imageId = new GuAmiParameter(scope, { app }),
       minimumInstances,
       maximumInstances,
+      requireImdsv2 = true,
```

Comment:
Having given it some thought, I don't think we should call this `requireImdsv2` as the AWS default is `false` and this will appear in intellisense, even though GuCDK will default it to `true`, which will be confusing.

`withoutImdsv2` might be better? We've precedent for `without*` in the library already [here](https://github.com/guardian/cdk/blob/7b3299e2cb98f64ac23018441c29f825d2f1dbcc/src/constructs/core/stack.ts#L29) and [here](https://github.com/guardian/cdk/blob/7b3299e2cb98f64ac23018441c29f825d2f1dbcc/src/constructs/iam/roles/instance-role.ts#L16).

Reply:
Yeah makes sense - I've pushed that up!